ui/k-info-panel/switch_info.kytos

```diff
@@ -61,7 +69,60 @@
        window.d3.json(endpoint, function(error, result) {
          self.flows = result[dpid].flows
        })
-     }
+     },
+     bt_add_metadata() {
+       var _this = this
+       let request = $.ajax({
+                        type: "POST",
+                        url: this.$kytos_server_api + "kytos/topology/v3/switches/" + this.metadata.dpid
+                              + "/metadata",
+                        async: true,
+                        data: this.to_add,
+                        dataType: "json",
+                        contentType: "application/json; charset=utf-8",
+       });
+       request.done(function() {
+         let notification = {
+              icon: 'gear',
+              title: 'Add metadata: Success',
+              description: '"' + _this.to_add + '" was added to the metadata. Switch: ' + _this.metadata.id,
+         }
+         _this.$kytos.$emit("setNotification", notification)
+       });
+       request.fail(function(data) {
+         let notification = {
+              icon: 'gear',
+              title: 'Add metadata: Failure',
+              description: data.status + ': ' + data.responseJSON.description + ' "' + _this.to_add + '" was not added to the metadata. Switch: ' + _this.metadata.id,
+         }
+         _this.$kytos.$emit("setNotification", notification)
+       });
+     },
+     bt_rmv_metadata() {
+       var _this = this
+       let request = $.ajax({
+                        type: "DELETE",
+                        url: this.$kytos_server_api + "kytos/topology/v3/switches/" + this.metadata.dpid
+                              + "/metadata/" + this.to_delete,
+                        async: true,
+       });
+       request.done(function() {
+         let notification = {
+              icon: 'gear',
+              title: 'Delete metadata: Success',
+              description: '"' + _this.to_delete + '" was deleted from the metadata. Switch: ' + _this.metadata.id,
+         }
+         _this.$kytos.$emit("setNotification", notification)
+       });
+       request.fail(function(data) {
+         let notification = {
+              icon: 'gear',
+              title: 'Delete metadata: Failure',
+              description: data.status + ': ' + data.responseJSON.description + ' "' + _this.to_delete + '" was not deleted from the metadata. Switch: ' + _this.metadata.id,
+         }
+         _this.$kytos.$emit("setNotification", notification)
+       });
+     },
```

Comment:
@Alopalao, 

I wonder if we could align the buttons in a column oriented manner, just so the layout fits nicely, something like this:

![2022-03-10-122444_418x170_scrot](https://user-images.githubusercontent.com/1010796/157694648-8bf66283-2dfc-4875-8981-28e07f4a5f98.png)


Currently it display like this:
![2022-03-10-122529_410x162_scrot](https://user-images.githubusercontent.com/1010796/157694667-80b9f03b-8493-4084-b690-b1e9130b1218.png)

This CSS here would be a start, feel free to explore and apply it:

```
diff --git a/ui/k-info-panel/switch_info.kytos b/ui/k-info-panel/switch_info.kytos
index c723415..ec77972 100644
--- a/ui/k-info-panel/switch_info.kytos
+++ b/ui/k-info-panel/switch_info.kytos
@@ -20,10 +20,12 @@
          <k-flow :content="flow" :key="flow.id" v-for="flow in this.flows"></k-flow>
       </k-accordion-item>
       <k-accordion-item title="Metadata actions">
-         <k-input title="Insert metadata" icon="arrow-right" placeholder='Eg. {"bandwidth": 100, "port_name": "some_name"}' :value.sync="to_add"></k-input>
-         <k-button title="Add metadata" :on_click="bt_add_metadata"></k-button>
-         <k-input title="Insert metadata" icon="arrow-right" placeholder="Eg. port_name" :value.sync="to_delete"></k-input>
-         <k-button title="Remove metadata" :on_click="bt_rmv_metadata"></k-button>
+         <div class="metadata-container">
+           <k-input title="Insert metadata" icon="arrow-right" placeholder='Eg. {"bandwidth": 100, "port_name": "some_name"}' :value.sync="to_add"></k-input>
+           <k-button title="Add metadata" :on_click="bt_add_metadata"></k-button>
+           <k-input title="Insert metadata" icon="arrow-right" placeholder="Eg. port_name" :value.sync="to_delete"></k-input>
+           <k-button title="Remove metadata" :on_click="bt_rmv_metadata"></k-button>
+         </div>
       </k-accordion-item>
     </k-accordion>
 </template>
@@ -137,3 +139,9 @@
  }
 </script>

+<style>
+.metadata-container {
+  display: flex;
+  flex-direction: column;
+}
+</style>
```

Reply:
Changed on commit [8971b6a11225230e4703a953e2117093ec3e70d2](https://github.com/kytos-ng/topology/pull/74/commits/8971b6a11225230e4703a953e2117093ec3e70d2)

Reply:
@Alopalao appreciated this commit, could you adapt the CSS to look like the second more centered example you gave on this comment https://github.com/kytos-ng/topology/pull/74#issuecomment-1064313207? That one I think ended up looking better

Reply:
Updated on commit [948fff438820ac5e2bc198549045e0de39cc3f44](https://github.com/kytos-ng/topology/pull/74/commits/948fff438820ac5e2bc198549045e0de39cc3f44)